Sources/Proxy/Core/LookupTableProxy/index.js

```diff
@@ -3,7 +3,45 @@ import macro from 'vtk.js/Sources/macro';
 import vtkColorMaps from 'vtk.js/Sources/Rendering/Core/ColorTransferFunction/ColorMaps';
 import vtkColorTransferFunction from 'vtk.js/Sources/Rendering/Core/ColorTransferFunction';
 
-const DEFAULT_PRESET_NAME = 'Cool to Warm';
+import Constants from './Constants';
+
+const { Mode, Defaults } = Constants;
+
+// ----------------------------------------------------------------------------
+
+function applyPresetToLookupTable(presetName, dataRange, lut) {
+  lut.applyColorMap(vtkColorMaps.getPresetByName(presetName));
+  lut.setMappingRange(dataRange[0], dataRange[1]);
+  lut.updateRange();
+}
+
+// ----------------------------------------------------------------------------
+
+function applyRGBPointsToLookupTable(rgbPoints, dataRange, lut) {
+  const width = dataRange[1] - dataRange[0];
+  const rescaled = rgbPoints.map(([x, ...rgb]) => [
+    x * width + dataRange[0],
+    ...rgb,
+  ]);
+
```

Comment:
setMappingRange is doing the rescale. So no need to do it.

Reply:
Oh haha I thought it was going to be the same as with PiecewiseFunction...